pyproject.toml

```diff
@@ -26,18 +26,25 @@ keywords = [
 classifiers = [
   "Development Status :: 4 - Beta",
   "Intended Audience :: Developers",
-  "Natural Language :: English",
+  "Topic :: Software Development",
   "Framework :: Django",
-  "Framework :: Django :: 4.0",
-  "Framework :: Django :: 4.1",
   "Framework :: Django :: 4.2",
   "Framework :: Django :: 5.0",
+  "Framework :: Django :: 5.1",
```

Comment:
![image](https://github.com/user-attachments/assets/4730d89f-ec73-4fc2-9321-9d42e12d4c14)
```suggestion
  "Framework :: Django :: 5.1",
  "Framework :: Django :: 5.2",
```

Reply:
Opps, my bad django 5.2 only expected in April 2025